pallets/reward-distribution/src/benchmarking.rs

```diff
@@ -39,15 +39,11 @@ pub mod benchmarks {
 			collateral_currency,
 		);
 
-		let _stake = T::VaultStaking::deposit_stake(&vault_id, &nominator, nominated_amount.into())
```

Comment:
The trait `Staking` in `pallet-staking` returns `Result<(), DispatchError>`; so no need for _`let <variable_name>`_.